tools/setup.py

```diff
@@ -0,0 +1,107 @@
+import subprocess
+import os
+import platform
+from tools.colors import Colors
+
+class SceSetupTool:
+    """
+    This class handles checking installation of the proper tools
+    as well as checking if the proper directories are cloned for
+    sce development
+    """
+    os = ""
+    color = Colors()
+
+    def check_installation(self, name, command, link):
+        """
+        This method is called to check if the proper software is installed
+        if the software is installed the console will print out a message and move on
+        if not then the console will redirect the user to the site to install the softare
+            Parameters:
+            name (string): name of the software to check for
+            command (string): the name of the command that the command line wiill execute
+            link (string): link to the site to install the software
+        """
+        devnull = open(os.devnull, 'wb')
+        software = subprocess.check_call(command, stdout=devnull, stderr=subprocess.STDOUT)
+        if software == 0:
+            self.color.print_yellow(name + " found", True)
+        else:
+            self.color.print_red(name + " not found", True)
+            print("visit here to install: ")
+            self.color.print_purple(link, True)
+            print("press enter to continue: ")
+            choice = input()
+            while choice != "":
+                choice = input()
+
+    def check_os(self):
+        """
+        This method checks the user's os and stores it into a class variable
+        """
+        self.os = platform.system()
+        self.color.print_purple(self.os, True)
+
+    def check_directory(self, name):
```

Comment:
lets hold onto this function, it can be a part of a different command called `sync` or something, e.g.

```sh
# updates each repo's main branch
sce sync
# updates a given repo's main branch
sce sync --repo <repo name>
```

Reply:
in other words u can remove it for now, and open an issue called "Add sync script" and we can paste the code for updating a branch there.